starters/serverless-apollo-contentful/README.md

```diff
@@ -167,6 +167,26 @@ How to run migrations:
 
 To pre-populate the data within contentful we implement seeding scripts which are located in `scripts/seed`.
 
+### Queueing
+
+Amazon Simple Queue Service (SQS) is a fully managed message queuing service that enables the decoupling of distributed applications and microservices and ensures reliable and scalable processing of application messages.
+
+In this implementation, an SQS message is sent by the handler `sqs-generate-job` via the `sendMessage` function, which is then stored in a queue called `DemoJobQueue`. The SQS handler `sqs-handler` polls this queue and processes any message received.
+
+The handler function of `sqs-handler` retrieves the messages from the event parameter passed to it, calls the `recordHandler` function for each record in the event, and logs the message.
+
+The `recordHandler` function processes the individual message by taking the body of the message and logging it.
```

Comment:
Should we add something here to indicate that the logging is just for demo purposes?

e.g.,
> The demo implementation just logs the body to showcase the message successfully arrived at the handler. A real-world application would parse the message and act on it.